public/keymaps/b/basekeys_slice_rev1_default.json

```diff
@@ -0,0 +1,22 @@
+{
+  "keyboard": "basekeys/slice/rev1",
+  "keymap": "default_4704aba",
+  "commit": "4704aba55cac8ee6d382498aafac9e61e48bfadf",
+  "layout": "LAYOUT",
+  "layers": [
+    [
+      "KC_ESC",    "KC_GRV",    "KC_1",    "KC_2",    "KC_3",    "KC_4",    "KC_5",        "KC_6",    "KC_7",    "KC_8",    "KC_9",    "KC_0",    "KC_MINS",    "KC_EQL",        "KC_BSPC",
+      "KC_MPLY",   "KC_TAB",    "KC_Q",    "KC_W",    "KC_E",    "KC_R",    "KC_T",        "KC_Y",    "KC_U",    "KC_I",    "KC_O",    "KC_P",    "KC_LBRC",    "KC_RBRC",    "KC_BSLS",
+      "KC_MNXT",   "KC_CAPS",   "KC_A",    "KC_S",    "KC_D",    "KC_F",    "KC_G",       "KC_H",    "KC_J",    "KC_K",    "KC_L",    "KC_SCLN",    "KC_QUOT",                   "KC_ENT",
+      "KC_MPRV",   "KC_LSFT",   "KC_Z",    "KC_X",    "KC_C",    "KC_V",    "KC_B",        "KC_B",    "KC_N",    "KC_M",    "KC_COMM",    "KC_DOT",    "KC_SLSH",    "KC_RSFT",    "KC_RSFT",
+      "KC_MUTE",   "KC_LCTL",  "KC_LALT",    "KC_SPC",    "KC_SPC",                        "KC_SPC",    "KC_SPC",                                      "KC_RALT",    "KC_RCTL",    "MO(1)"
+    ],
+    [
+      "KC_ESC",    "KC_NO",    "KC_F1",    "KC_F2",    "KC_F3",    "KC_F4",    "KC_F5",    "KC_F6",    "KC_F7",    "KC_F8",    "KC_F9",    "KC_F10",    "KC_F11",    "KC_F12",    "KC_DEL",
+      "KC_NO",    "KC_NO",    "KC_NO",    "KC_NO",    "KC_NO",    "KC_NO",    "KC_NO",    "KC_NO",    "KC_NO",    "KC_PSCR",    "KC_SLCK",    "KC_PAUS",    "KC_UP",    "KC_NO",    "KC_NO",
+      "KC_NO",    "KC_NO",    "KC_NO",    "KC_NO",    "KC_NO",    "KC_NO",    "KC_NO",    "KC_NO",    "KC_HOME",    "KC_PGUP",    "KC_LEFT",    "KC_RGHT",    "KC_NO",    "KC_NO",
+      "KC_NO",    "KC_NO",    "KC_NO",    "KC_NO",    "KC_NO",    "KC_NO",    "KC_NO",    "KC_NO",    "KC_END",    "KC_PGDN",    "KC_DOWN",    "KC_NO",    "KC_NO",    "KC_NO",    "KC_NO",
+      "KC_NO",    "KC_NO",    "KC_NO",    "KC_NO",    "KC_NO",    "KC_NO",    "KC_NO",    "KC_NO",    "KC_NO",    "KC_TRNS"
+    ]
```

Comment:
This should be as close as possible to what the default keymap on qmk_firmware does.

```suggestion
    [
      "KC_ESC",  "KC_GRV",  "KC_1",    "KC_2",    "KC_3",    "KC_4",    "KC_5",    "KC_6",    "KC_7",    "KC_8",    "KC_9",    "KC_0",    "KC_MINS", "KC_EQL",  "KC_BSPC",
      "RGB_MOD", "KC_TAB",  "KC_Q",    "KC_W",    "KC_E",    "KC_R",    "KC_T",    "KC_Y",    "KC_U",    "KC_I",    "KC_O",    "KC_P",    "KC_LBRC", "KC_RBRC", "KC_BSLS",
      "RGB_M_G", "KC_CAPS", "KC_A",    "KC_S",    "KC_D",    "KC_F",    "KC_G",    "KC_H",    "KC_J",    "KC_K",    "KC_L",    "KC_SCLN", "KC_QUOT",            "KC_ENT",
      "RGB_M_X", "KC_LSFT", "KC_Z",    "KC_X",    "KC_C",    "KC_V",    "KC_B",    "KC_B",    "KC_N",    "KC_M",    "KC_COMM", "KC_DOT",  "KC_SLSH", "KC_RSFT", "KC_RSFT",
      "RGB_TOG", "KC_LCTL", "KC_LALT",                       "KC_LALT", "KC_SPC",  "KC_SPC",  "KC_SPC",                                   "KC_RALT", "KC_RCTL", "MO(1)"
    ],
    [
      "KC_ESC",  "TG(2)",   "KC_F1",   "KC_F2",   "KC_F3",   "KC_F4",   "KC_F5",   "KC_F6",   "KC_F7",   "KC_F8",   "KC_F9",   "KC_F10",  "KC_F11",  "KC_F12",  "KC_BSPC",
      "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_PSCR", "KC_SLCK", "KC_PAUS", "KC_UP",   "KC_TRNS", "KC_TRNS", "KC_TRNS",
      "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_HOME", "KC_PGUP", "KC_LEFT", "KC_RGHT", "KC_TRNS",            "KC_TRNS",
      "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_END",  "KC_PGDN", "KC_DOWN", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS",
      "KC_TRNS", "KC_TRNS", "KC_TRNS",                       "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS",                                  "KC_TRNS", "KC_TRNS", "KC_TRNS"
    ],
    [
      "KC_NO",   "TG(2)",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "RESET",
      "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",
      "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "RGB_TOG", "RGB_MOD", "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",              "KC_NO",
      "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",   "RGB_VAD", "RGB_VAI", "RGB_HUD", "RGB_HUI", "RGB_SAD", "RGB_SAI", "KC_NO",   "KC_NO",
      "KC_NO",   "KC_NO",   "KC_NO",                         "KC_NO",   "KC_NO",   "KC_NO",   "KC_NO",                                    "KC_NO",   "KC_NO",   "KC_NO"
    ]
```